code/game/mecha/equipment/weapons/weapons.dm

```diff
@@ -151,6 +151,25 @@
 		return 1
 	return 0
 
+//Exosuit-mounted kinetic accelerator
+/obj/item/mecha_parts/mecha_equipment/weapon/energy/mecha_kineticgun
+	equip_cooldown = 10
+	name = "Exosuit Proto-kinetic Accelerator"
+	desc = "An exosuit-mounted mining tool that does increased damage in low pressure. Drawing from an onboard power source allows it to project further than the handheld version."
+	icon_state = "mecha_kineticgun"
+	energy_drain = 30
+	projectile = /obj/projectile/kinetic/mech
+	fire_sound = 'sound/weapons/kenetic_accel.ogg'
+	harmful = TRUE
+
+//attachable to all mechas, like the plasma cutter
+/obj/item/mecha_parts/mecha_equipment/weapon/energy/mecha_kineticgun/can_attach(obj/mecha/working/M)
+	if(..()) //combat mech
+		return 1
+	else if(M.equipment.len < M.max_equip && istype(M))
+		return 1
+	return 0
```

Comment:
use return TRUE and return FALSE

Reply:
and a descriptive variable name, not M

Reply:
I think the M goes back way further than right there.

Reply:
https://github.com/shiptest-ss13/Shiptest/blob/master/.github/CONTRIBUTING.md#use-self-explanatory-var-names

Reply:
From what I can tell even with ports the ideal is to follow our contribution guidelines to have decently readable and modifiable code, including avoiding single character variable naming even when it seems pointless or is the standard elsewhere

Reply:
alright